src/pages/transaction-signing/components/actions.tsx

```diff
@@ -123,12 +124,12 @@ const SubmitAction = (props: ButtonProps) => {
 const FeeRowItemSuspense = () => {
   let showWarning = false;
   const defaultFee = useCurrentDefaultFee();
+  const customFee = useTransactionRequestCustomFee();
   const transactionRequest = useTransactionRequest();
   const appName = transactionRequest?.appDetails?.name;
-  const customFee = transactionRequest?.fee;
 
   if (!!customFee && defaultFee) {
-    showWarning = customFee > defaultFee.toNumber() * 4;
+    showWarning = customFee.isGreaterThan(defaultFee.times(4));
```

Comment:
This is a magic number, see it was there before but a long, descriptive variable makes code much much easier to understand.

```suggestion
    const multipleFromWhichFeeIsConsideredLarge = 4;
    showWarning = customFee.isGreaterThan(defaultFee.times(multipleFromWhichFeeIsConsideredLarge));
```